.github/workflows/test-batch-processing.yml

```diff
@@ -121,8 +121,12 @@ jobs:
 
       - name: Diff summaries (Ubuntu vs macOS)
         continue-on-error: true
-        run: diff -u artifacts/sct_example_data_ubuntu-20.04.json artifacts/sct_example_data_macos-13.json | ydiff
+        run: diff -u artifacts/sct_example_data_ubuntu-22.04.json artifacts/sct_example_data_macos-13.json | ydiff
 
       - name: Diff summaries (Ubuntu vs. Windows)
         continue-on-error: true
-        run: diff -u artifacts/sct_example_data_ubuntu-20.04.json artifacts/sct_example_data_windows-2022.json | ydiff
\ No newline at end of file
+        run: diff -u artifacts/sct_example_data_ubuntu-22.04.json artifacts/sct_example_data_windows-2022.json | ydiff
+
+      - name: Diff summaries (Ubuntu 22 vs. Ubuntu 24)
```

Comment:
Cool!

Reply:
Thanks for reminding me about this! 

It looks like there's no difference between 22 and 24? The only difference is in the [hash of the output CSV files](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/13290160574/job/37111319384#step:6:16), but I think this is only because the CSV files contain timestamps?

I'd be curious to maybe remove those timestamps to see if we can minimize the diff (and thus only highlight important diffs).